go/stats/export.go

```diff
@@ -247,3 +252,42 @@ func stringMapToString(m map[string]string) string {
 	fmt.Fprintf(b, "}")
 	return b.String()
 }
+
+var (
+	dimMu             sync.Mutex
+	droppedDimensions map[string]bool
+)
+
+// IsDimensionDropped returns true if the specified dimension is dropped.
+func IsDimensionDropped(name string) bool {
+	dimMu.Lock()
+	defer dimMu.Unlock()
+
+	if droppedDimensions == nil {
+		dims := strings.Split(*dropDimensions, ",")
+		droppedDimensions = make(map[string]bool, len(dims))
+		for _, dim := range dims {
+			droppedDimensions[dim] = true
+		}
+	}
+	return droppedDimensions[name]
+}
+
+// safeJoinLabels joins the label values with ".", but first replaces any existing
+// "." characters in the labels with the proper replacement, to avoid issues parsing
+// them apart later.
```

Comment:
Please include some description here about how it also drops (or better "combines") values from the named dropped dimensions.